src/main/resources/META-INF/plugin.xml

```diff
@@ -9,7 +9,14 @@
 
     <change-notes>
         <![CDATA[
-          <li> The plugin cannot find the NPM executable in some cases.</li>
+          <li> Add 'Show in dependencies tree' button in the package descriptors.</li>
```

Comment:
...in the package descriptors. --> ...in the editor, while clicking on a dependency.